### PR TITLE
First commit Casptone Project

### PR DESCRIPTION
**Pull request by ppmarq1**
Hi! I kindly ask you for a review of this project. I've completed this milestone where we were required to:

To build the 2 pages, as follow:
The home page.
The about page.

Each of these pages has versions for 2 different screen sizes:

Mobile: up to 768px wide.
Desktop: 768px or wider.

Interactions

Links
The home page has a link in the menu to the about page.
The logo in the header links to the home page.

Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
The mobile menu has a close (X) button that closes the menu.

Dynamic page

The section "Featured speakers" was created dynamically in JavaScript.

The Original design idea by Cindy Shin in Behance.

Project documentation

Please see the Demo video with a description of the project (please click on the link below), including all the project features (different pages and different screen sizes).

https://www.loom.com/share/b5e4fff7c5f44e65ac63c66e869bc514

Best Regards,

Pedro Marques

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (micheaol) [inline index.html:6]: Please ensure that you make use of GitHub Pages to deploy the web pages and a link to the deployed project is present in the README according to the project requirement.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      <div class="logo">
+        <a href="index.html"
+          ><img src="images/Logo/logo.avif" alt="logo"   
+        /></a>
+      </div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#Contact">News</a></li>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+      </section>
+      <footer id="home-footer">
+        <div id="footer-img">
+          <img src="images/Logo/logo.avif" alt="footer logo" />
+        </div>
+        <div id="footer-text">
+          <p>© 2022 Collision Inc. All Rights Reserved</p>
+          <p></p>
+        </div>
+      </footer>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+          <h2>WEB SUMMIT 2023</h2>
+          <p id="introduction">
+            Although times may be uncertain, one thing
+            remains true. There is a simple power in people
+            coming together.
+          </p>
+          <h3>2023.07.24(MON) ~ 28(FRI)</h3>
+          <p>ENNERCARE CENTRE, TORONTO - CANADA.</p>
+        </div>
+      </section>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            <h2>Lecture</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program2.png" alt="Exhibition" />
+            <h2>Exhibition</h2>
+            <p>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design, the fonts, and also please implement the `hover effect` on the cards).

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+        <button type="button" id="more-button">MORE</button>
+      </section>
+      <section id="home-partners">
+        <h2>Partners</h2>
+        <hr class="solid" />
+        <div id="partner-img">
+          <img src="images/partners/logo-accenture.png" alt="accenture" />
+          <img src="images/partners/cisco-logo.png" alt="cisco" />
+          <img src="images/partners/kpmg-logo.png" alt="kpmg" />
+        </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+            <h2>Forum</h2>
+            <p>
+              Have the time to share your thoughts and opinions with experts
+              for each topic.
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program4.png" alt="Workshop" />
+            <h2>Workshop</h2>
+            <p>
```
Comment (micheaol) [inline index.html:102]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts sizes).

Your Implementation | Design
:---:|:---:
image | image

Your Implementation | Design
:---:|:---:
image | image

Your Implementation | Design
:---:|:---:
image | image

File: about.html [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+              politicians and cultural figureheads who are
+              reshaping the world.</span
+            >
+          </p>
+          <h3>
+            Please contact us per email for any further questions about the WEB SUMMIT 2023!
+          </h3>
+          <p id="email">info@torontosummit.com</p>
+        </div>
+      </section>
```
Comment (micheaol) [inline about.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image

File: about.html [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+      <section id="logo-reveal">
+        <h2>WEB SUMMIT 2023</h2>
+        <hr class="solid" />
+        <p>
+          The Global Summit brings together the community of experts and academics,
+          who compromise Collisions affiliate network in a different country every 
+          two years. This year, we hope to extend our invitation list including 
+          organisation, individuals who want to work with us on shared projects that
```
Comment (micheaol) [inline about.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image

File: about.html [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+        </div>
+      </section>
+      <section id="past-summits">
+        <h2>See the past WEB Summits</h2>
+        <hr class="solid" />
+        <p>
+          Take a look at the last two WEB Summits whick took place
+          in Lisbon and Dublin
+        </p>
+        <div id="past-pair">
```
Comment (micheaol) [inline about.html]: Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
image | image


Your Implementation | Design
:---:|:---:
image | image


Your Implementation | Design
:---:|:---:
image | image

File: main.css [hunk 11]
```diff
@@ -0,0 +1,800 @@
...
+  margin: 0;
+  overflow-x: hidden;
+}
+
+#back {
+  background-repeat: repeat;
+  position: absolute;
+  opacity: 0.1;
+}
+
```
Comment (micheaol) [inline main.css:13]: Please ensure that all page elements have similar spacing, borders, shadows, and paddings to the original design.

File: index.js [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+  },
+  {
+    name: 'Michelle Zatlyn',
+    image: 'images/speakers/Michelle.avif',
+    graduate: 'Co-founder Cloudflare',
+    experience: 'Michelle is co-founder and COO of Cloudflare,one of the world’s largest networks',
+  },
+];
+
+// console.log(speakersList)
```
Comment (micheaol) [inline index.js]: Please remove `console.log` from your codebase. It's a good practice to remove `console.log` from the codebase and also for security reasons.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ppmarq1 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ppmarq1) [reply index.html:6]: Github Pages implemented.

File: index.html (hunk 2 above)
Comment (ppmarq1) [reply index.html]: resolved

File: index.html (hunk 3 above)
Comment (ppmarq1) [reply index.html]: resolved

File: index.html (hunk 4 above)
Comment (ppmarq1) [reply index.html]: resolved

File: about.html (hunk 9 above)
Comment (ppmarq1) [reply about.html]: resolved

File: index.js (hunk 12 above)
Comment (ppmarq1) [reply index.js]: resolved

File: index.html (hunk 5 above)
Comment (ppmarq1) [reply index.html]: resolved

File: index.html (hunk 6 above)
Comment (ppmarq1) [reply index.html]: resolved

File: about.html (hunk 8 above)
Comment (ppmarq1) [reply about.html]: resolved

File: about.html (hunk 10 above)
Comment (ppmarq1) [reply about.html]: resolved

File: index.html (hunk 7 above)
Comment (ppmarq1) [reply index.html:102]: resolved

File: main.css (hunk 11 above)
Comment (ppmarq1) [reply main.css:13]: resolved

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @ppmarq1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- All Requirements have been well set 👏 👏 👏 
- Great Job 👏 👏 
## Required Changes ♻️

- [x] Please Fix the broken images. 



- [ ] 

f1

- [x] Another image




f2






## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @ppmarq1.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**